server/src/test/java/org/elasticsearch/index/query/QueryStringQueryBuilderTests.java

```diff
@@ -382,10 +383,11 @@ public QueryStringQueryBuilder mutateInstance(QueryStringQueryBuilder instance)
     protected void doAssertLuceneQuery(QueryStringQueryBuilder queryBuilder,
                                        Query query, SearchContext context) throws IOException {
         assertThat(query, either(instanceOf(TermQuery.class))
-            .or(instanceOf(BooleanQuery.class)).or(instanceOf(DisjunctionMaxQuery.class))
-            .or(instanceOf(PhraseQuery.class)).or(instanceOf(BoostQuery.class))
-            .or(instanceOf(MultiPhrasePrefixQuery.class)).or(instanceOf(PrefixQuery.class)).or(instanceOf(SpanQuery.class))
-            .or(instanceOf(MatchNoDocsQuery.class)));
+                .or(instanceOf(BooleanQuery.class)).or(instanceOf(DisjunctionMaxQuery.class))
+                .or(instanceOf(BlendedTermQuery.class)).or(instanceOf(PhraseQuery.class))
+                .or(instanceOf(BoostQuery.class)).or(instanceOf(MultiPhrasePrefixQuery.class))
+                .or(instanceOf(PrefixQuery.class)).or(instanceOf(SpanQuery.class))
+                .or(instanceOf(MatchNoDocsQuery.class)));
```

Comment:
+1 to remove these assertions, the `query_string` query can build all sort of queries so I don't think we should track the full list here. This will also allow more flexibility in the random query generator.